crates/cxx-qt-gen/src/parser/method.rs

```diff
@@ -102,4 +110,90 @@ impl ParsedMethod {
             name,
         })
     }
+
+    /// Returns a [TokenStream] of the methods parameters, which can be used in generation stage
+    /// class_name is the rust class name which this method is defined for
+    pub fn get_params_tokens(&self, class_name: &Ident) -> TokenStream {
```

Comment:
wonder if this method should be in the parser or generator :') Maybe it's fine here though :thinking:

Reply:
in generator/mod.rs now